src/ImageRunModal.jsx

```diff
@@ -687,7 +687,44 @@ export class ImageRunModal extends React.Component {
                 <Tabs activeKey={activeTabKey} onSelect={this.handleTabClick}>
                     <Tab eventKey={0} title={<TabTitleText>{_("Details")}</TabTitleText>} className="pf-c-form pf-m-horizontal">
                         { this.props.userServiceAvailable && this.props.systemServiceAvailable &&
-                        <FormGroup isInline hasNoPaddingTop fieldId='run-image-dialog-owner' label={_("Owner")}>
+                        <FormGroup isInline hasNoPaddingTop fieldId='run-image-dialog-owner' label={_("Owner")}
+                                   labelIcon={
+                                       <Popover aria-label={_("Owner help")}
+                                          enableFlip
+                                          bodyContent={
+                                              <>
+                                                  <TextContent>
+                                                      <Text component={TextVariants.h4}>{_("System")}</Text>
+                                                      <TextList>
+                                                          <TextListItem>
+                                                              {_("Ideal for running services")}
+                                                          </TextListItem>
+                                                          <TextListItem>
+                                                              {_("Resource limits can be set")}
+                                                          </TextListItem>
+                                                          <TextListItem>
+                                                              {_("Checkpoint/restore support")}
+                                                          </TextListItem>
+                                                          <TextListItem>
+                                                              {_("Ports under 1024 can be forwarded")}
```

Comment:
Can't you use ports under 1024 inside the container and forward to ports 1024 outside? This sounds like the port _inside_ the container must be >= 1024.

Perhaps:

```suggestion
                                                              {_("Ports under 1024 can be mapped")}
```

Reply:
Yes, it's only on the outside so podman run -p 22:22 only works as root.